war/src/main/scss/base/spacing.scss

```diff
@@ -0,0 +1,49 @@
+$unit: 0.4rem;
+
+@for $i from 0 through 6 {
+  .jenkins-\!-margin-#{$i} {
+    margin: $i * $unit !important;
+  }
+
+  .jenkins-\!-margin-top-#{$i} {
+    margin-top: $i * $unit !important;
+  }
+
+  .jenkins-\!-margin-left-#{$i} {
+    margin-left: $i * $unit !important;
+  }
+
+  .jenkins-\!-margin-bottom-#{$i} {
+    margin-bottom: $i * $unit !important;
+  }
+
+  .jenkins-\!-margin-right-#{$i} {
+    margin-right: $i * $unit !important;
+  }
+}
+
+.jenkins-\!-margin-auto {
+  margin: 0 auto;
+}
+
+@for $i from 1 through 6 {
```

Comment:
Why doesn't this include 0? The following styles from the old compiled CSS are now gone:

```css
.jenkins-\!-padding-0 {
  padding: 0rem !important;
}
.jenkins-\!-padding-top-0 {
  padding-top: 0rem !important;
}
.jenkins-\!-padding-left-0 {
  padding-left: 0rem !important;
}
.jenkins-\!-padding-bottom-0 {
  padding-bottom: 0rem !important;
}
.jenkins-\!-padding-right-0 {
  padding-right: 0rem !important;
}
```

If this was intentional, then please disregard this comment.

Reply:
Fixed now :)